srcpkgs/passwordsafe/patches/use-gtest-from-package.patch

```diff
@@ -0,0 +1,37 @@
+--- CMakeLists.txt.ORIG	2021-02-25 23:45:37.917034534 +0100
++++ CMakeLists.txt	2021-02-26 00:05:34.138397928 +0100
+@@ -255,34 +255,6 @@
+ endif (USE_ASAN)
+ endif (MSVC)
+ 
+-if (NOT NO_GTEST)
```

Comment:
Maybe we can get a flag to disable this upstream?

Reply:
yep, I will request.